appstudio-utils/util-scripts/check-ec-policy.sh

```diff
@@ -8,7 +8,7 @@ POLICY_DIR=$EC_WORK_DIR/policies
 
 FORMAT=${1:-pretty}
 
-OPA_QUERY=${OPA_QUERY:-data.hacbs.contract.main.deny}
+OPA_QUERY=${OPA_QUERY:-data.main.deny}
```

Comment:
It makes me wonder if we should introduce OPA_QUERY as a task parameter. Probably not at this point...